server-spi-private/src/main/java/org/keycloak/organization/OrganizationProvider.java

```diff
@@ -56,10 +57,26 @@ public interface OrganizationProvider extends Provider {
     OrganizationModel getByDomainName(String domainName);
 
     /**
-     * Returns the organizations of the given realm as a stream.
-     * @return Stream of the organizations. Never returns {@code null}.
+     * Returns all organizations in the realm.
+     *
+     * @return a {@link Stream} of the realm's organizations.
+     */
+    default Stream<OrganizationModel> getOrganizations() {
```

Comment:
Like other methods, we are avoiding redundancy in names by not using organization in names. For me, `getAll` sounds better.

Reply:
Same thing - tried to follow what we already do in other providers. But I have changed it to `getAllStream` for both methods in the provider - after all the only difference is in the params, so the name should be the same.